File: src/main/scala-graphql/io/github/graphql/scala/model/DismissPullRequestReviewMutationResponse.scala
package io.github.graphql.scala.model

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResult
import java.util.{ Map => JMap }

@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2021-03-07T19:49:37+0800"
)
class DismissPullRequestReviewMutationResponse extends GraphQLResult[JMap[String, DismissPullRequestReviewPayloadTO]] {

    def dismissPullRequestReview(): DismissPullRequestReviewPayloadTO = {
        val data: JMap[String, DismissPullRequestReviewPayloadTO] = getData
        if (data != null) data.get(DismissPullRequestReviewMutationResponse.OPERATION_NAME) else null
    }

}

object DismissPullRequestReviewMutationResponse {

    private final val OPERATION_NAME: String = "dismissPullRequestReview"

}

File: src/main/java-graphql/io/github/graphql/model/DismissPullRequestReviewMutationResponse.java
package io.github.graphql.model;

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResult;
import java.util.Map;

@javax.annotation.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-03-06T19:47:48+0800"
)
public class DismissPullRequestReviewMutationResponse extends GraphQLResult<Map<String, DismissPullRequestReviewPayloadTO>> {

    private static final String OPERATION_NAME = "dismissPullRequestReview";

    public DismissPullRequestReviewMutationResponse() {
    }

    public DismissPullRequestReviewPayloadTO dismissPullRequestReview() {
        Map<String, DismissPullRequestReviewPayloadTO> data = getData();
        return data != null ? data.get(OPERATION_NAME) : null;
    }

}
